post-install-setup.sh

```diff
@@ -0,0 +1,237 @@
+#  modification, are permitted provided that the following conditions are met:#
+#                                                                             #
+#  - Redistributions of source code must retain the above copyright notice,   #
+#      this list of conditions and the following disclaimer.                  #
+#                                                                             #
+#  - Redistributions in binary form must reproduce the above copyright        #
+#      notice, this list of conditions and the following disclaimer in the    #
+#      documentation and/or other materials provided with the distribution.   #
+#                                                                             #
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"#
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE  #
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE #
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE  #
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR        #
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF       #
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS   #
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN    #
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)    #
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF     #
+#  THE POSSIBILITY OF SUCH DAMAGE.                                            #
+# ----------------------------------------------------------------------------#
+#
+# Details:
+#	This program is intended to be run after the software is installed
+#	mainly as an install helper.  The "normal" way liota would be
+#	installed is via pip, or possibly git, assuming you don't get the
+#	software from your distro (which is *ALWAYS* the preferred method).
+#	This will run through and setup some directories and copy some
+#	config files around into the right places so that you can run this
+#	and just get running.
+#
+# Caveats:
+#	There are going to be a bunch of assumptions about how your local
+#	system is setup, and these might not ultimately be correct.
+#	This will modify your system, should be run as root (it's going to 
+#	create directories and such in places that a normal user would not
+#	have access to, and you should always be cautious in running this,
+#	or any software as root, at the best of times.
+#
+# ----------------------------------------------------------------------------
+# Things that are being done / todo list
+#	- Check for the users we'll be using
+#		- Check the root user
+#		- Check the liota user
+#		- Check the liota group
+#	- Configuration files: /etc/liota
+#		- If /etc/liota doesn't exist, make it
+#		- Change the owner and group to what we are expecting
```

Comment:
User can create and change the ownership manually as required by him. We can comment out the Liota user code and can use it later if we are allowed to create new "liota" user on the gateway. I can check and can get back by tomorrow.